src/DB/db.js

```diff
@@ -0,0 +1,9 @@
+exports.getEvents = {
+  id: 1,
+  title: 'Community Challenge',
+  eventLocation: 'Sirikwa Hotel',
+  startTime: '2019-08-15 21:05:15.723336+07',
+  endTime: '2019-08-15 21:05:15.723336+07',
+  media: 'https://photos.app.goo.gl/323EcdkmFTzrfLd96',
+  speakers: 'Marvin Kweyu'
```

Comment:
speakers should be an array of objects, each object containing speaker name, job title, bio.